components/browser/state/src/main/java/mozilla/components/browser/state/state/ContentState.kt

```diff
@@ -77,6 +80,7 @@ data class ContentState(
     val webAppManifest: WebAppManifest? = null,
     val firstContentfulPaint: Boolean = false,
     val history: HistoryState = HistoryState(),
+    val permissionHighlights: PermissionHighlightsState? = null,
```

Comment:
Do we need `null` to model "we have none" or could we just use `PermissionHighlightsState()` (with default values) which will make using this state much simpler.

Reply:
👍🏽